README.md

```diff
@@ -1,104 +1,72 @@
-# Kernel Containers
+<p align="center">
+  <img src="static/images/Kernel-Wordmark_Accent.svg" alt="Kernel Logo" width="55%">
+  <br/><br/>
+  Browsers, nicely packaged in containers
+  <br/><br/>
+</p>
 
-## Overview
+<p align="center">
+  <img alt="GitHub License" src="https://img.shields.io/github/license/onkernel/kernel-images">
+  <a href="https://discord.gg/FBrveQRcud"><img src="https://img.shields.io/discord/1342243238748225556?logo=discord&logoColor=white&color=7289DA" alt="Discord"></a>
+  <a href="https://x.com/juecd__"><img src="https://img.shields.io/twitter/follow/juecd__" alt="Follow @juecd__"></a>
+  <a href="https://x.com/rgarcia"><img src="https://img.shields.io/twitter/follow/rfgarcia" alt="Follow @rfgarcia"></a>
+</p>
 
-Kernel provides containerized, ready-to-use Chrome browser environments for agentic workflows that need to access the Internet. `containers/docker/Dockerfile` and `unikernels/unikraft-cu` are the core infra that powers our [hosted services](https://docs.onkernel.com/introduction).
+### 📜 Table of Contents
+- [Overview](#overview)
+- [Key Features](#key-features)
+- [What You Can Do With It](#what-you-can-do-with-it)
+- [Quickstarts](#quickstarts)
+- [Contributing](#contributing)
+- [License](#license)
+- [Support](#support)
 
-### Key Features
+### 🌰 Overview
+
+Kernel provides containerized, ready-to-use Chrome browser environments for agentic workflows that need to access the Internet. `containers/docker/Dockerfile` and `unikernels/unikraft-cu` are the core infra that powers our hosted services.
+
+🌟[__Sign-up for the waitlist__](https://onkernel.com)🌟
```

Comment:
Perhaps worth sending people to a signup or waitlist link directly, but idk how that will work, so up to you.